spec/models/purchase_contact_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe PurchaseContact, type: :model do
+  describe '商品購入機能_商品配送先登録' do
+    before do
+      user = FactoryBot.create(:user)
+      @purchase_contact = FactoryBot.build(:purchase_contact, user_id: user.id)
+    end
+
+    context '商品配送先登録できるとき' do
+      it 'すべての値が正しく入力されていれば保存できること' do
+        expect(@purchase_contact).to be_valid
+      end
```

Comment:
建物名が空でも保存できないことをテストしましょう！

Reply:
ありがとうございます。
今回の仕様上で建物名は任意入力だと思いますので、恐らく「建物名を入力しなくても保存できること」を追加すべきだと考えます。勘違いかもしれないが、下記のような認識で大丈夫でしょうか。
```suggestion    context '商品配送先登録できるとき' do
      it 'すべての値が正しく入力されていれば保存できること' do
        expect(@purchase_contact).to be_valid
      end
      it '建物名を入力しなくても保存できること' do
        @purchase_contact.building = ''
        expect(@purchase_contact).to be_valid
      end
    end
```

Reply:
コメントありがとうございます！
おっしゃるとおりです！
```
誤) 建物名が空でも保存できないことをテストしましょう！
↓↓↓
正) 建物名が空でも保存できることをテストしましょう！
```
私が作成した文章に誤りがございました！
修正いただきありがとうございます！

Reply:
ありがとうございました